go.mod

```diff
@@ -50,7 +50,7 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/okex/cosmos-sdk v0.39.3-0.20210128214757-49de8c8d67ef
+	github.com/cosmos/cosmos-sdk => github.com/okex/cosmos-sdk v0.39.3-0.20210203101013-94fc40572947
```

Comment:
make sure the order of merge:
1. merge cmsdk pr first;
2. update the commit with the result of latest merge;
3. merge okexchain pr.